chain-tx-enclave/tx-validation/enclave/src/lib.rs

```diff
@@ -1,42 +1,39 @@
-#![crate_name = "txvalidationenclave"]
-#![crate_type = "staticlib"]
```

Comment:
@yihuang have you actually tried running the outcome of building NON-mock version in SGX environment ?

Reply:
> @yihuang have you actually tried running the outcome of building NON-mock version in SGX environment ?

Yeah, it's buildable with image `baiduxlab/sgx-rust:1804-1.1.0`, or you can try `docker/Dockerfile`.

Reply:
buildable != correct ... I spent a countless of hours debugging situations where the enclave code was successfully "built", but mysteriously crashed or returned error codes... because of different issues that went wrong in the compilation process: e.g. did a syscall or called an instruction not allowed inside SGX env

Reply:
Yeah, I'll continue to test it after the issue 786.